src/test/java/gov/healthit/chpl/aqa/asserts/ListingHistoryEyeAsserts.java

```diff
@@ -0,0 +1,64 @@
+package gov.healthit.chpl.aqa.asserts;
+
+import static org.junit.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import cucumber.api.java.en.Then;
+import gov.healthit.chpl.aqa.model.ListingHistoryData;
+import gov.healthit.chpl.aqa.pageObjects.ListingHistoryEyePage;
+import gov.healthit.chpl.aqa.stepDefinitions.Base;
+
+/**
+ * Class ListingHistoryEyeAsserts definition.
+ */
+public class ListingHistoryEyeAsserts extends Base {
+
+    /**
+     * Assert that expected product edit details is displayed on product history eye.
```

Comment:
Comment says "expected product edit" when it could be any of three types of edit. Also says "product history eye" when we should be using "listing history eye" to ~avoid~ reduce confusion